### DISCUSSION TITLE
Does anyone got a "sick" transparent icon ?

### DISCUSSION THREAD

**Discussion by Theoyeah** [DEMOLITIONDON96/Demolition-Engine#31] [Source Q&A]
I wanna make a custom degraded color for it

**Comment by ThatOneSillyGuyDon** [comment]
> I wanna make a custom degraded color for it

since i made the art for the sick icon, i can send it to ya, got Discord or something i can send at?

**Reply by Theoyeah** [reply]
No, but you can just send it here (transparent)
